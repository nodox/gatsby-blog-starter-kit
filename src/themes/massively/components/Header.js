import React from 'react';
import Link from 'gatsby-link';

export default class Header extends React.Component {
  constructor(props) {
    super(props)
  }

  render() {
    return (
      <header id="header">
        <Link to="/" className="logo">{this.props.name}</Link>
      </header>
    );
  }
}
