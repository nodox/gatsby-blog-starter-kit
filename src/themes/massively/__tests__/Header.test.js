import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('<Header />', () => {
  const component = shallow(
    <Header />
  );

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });

});
