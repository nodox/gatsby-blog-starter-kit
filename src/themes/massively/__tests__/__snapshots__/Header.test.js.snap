// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<header
  id="header"
>
  <GatsbyLink
    className="logo"
    to="/"
  >
    Steven Natera
  </GatsbyLink>
</header>
`;
